### ISSUE TITLE
VSTS licensing for Azure AD Linked MPN Accounts 

### ISSUE THREAD

**Issue by pkskelly** [MicrosoftDocs/visualstudio-docs#850]
We recently went through connecting our Azure AD tenant to VSTS and this was still a VERY bumpy road.  We had to change the Azure subscription which was associated with the Azure AD directory and needed VSTS team support to unlink the subscription and link to a new subscription for billing.   Once the subscription associated with the directory issue was addressed, the documentation for linking Azure AD accounts to the correct license was extremely confusing (Content linking page below).  While the Alternate Identity information was a clue, there was nothing that explicitly stated how to assign licenses to the newly linked directory based accounts. 

In our specific situation as a Microsoft Partners we must allocate MSDN subscription licenses through the partner portal, and can only add MSA accounts through the Partner Portal, the documentation for linking the Azure AD Accounts to VSTS should be updated to explicitly state that the MSA account associated with your MPN based license MUST add your Azure AD identity alias (email address) as the Alternate Account.    

Perhaps an FAQ might cover this simply in order to save other partners some headaches.  

**Q: How do I associate Microsoft Partner Network (MPN) assigned licenses to Azure AD identities?** 
A: VSTS will recognize the license assigned to the original MSA account assigned a license in the Partner Portal.  Ensure that the MSA account has the MSDN Subscription privilege assigned in the Partner Portal.  

Once the MSDN subscription privilege is verified, perform the following steps for each users MSA account and associated Azure AD account.  

- Log into https://my.visualstudio.com using your MSA account (Hotmail, Outlook.com, Gmail, etc.) - **using an InPrivate or InCognito session is recommended**
- Once logged into https://my.visualstudio.com click on the Subscriptions menu item at the top of the page and you should have the following subscriptions listed:
```
Visual Studio Enterprise (MPN)
Visual Studio Dev Essentials
```
- Scroll to the bottom of the page
- Under the “Alternate account” section, enter your Azure AD `<tenant`>.com account (e.g. jsmith@contoso.com) and click “Save” or “Update” as needed.
- Log into https://contoso.visualstudio.com using your Contoso Azure AD account (contoso.com)

Once the MSA and Azure AD Accounts are configured to share the MPNB license, the VSTS Users page will appropriately display the correct Access Level.

Hopefully this helps someone else in the MPN category. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a0e967e1-0b8e-a6b5-13ef-bd206ca6d13a
* Version Independent ID: 612623f5-6252-6353-8c7c-c1be43bd8801
* Content: [Identities for Visual Studio subscribers](https://docs.microsoft.com/en-us/visualstudio/subscriptions/vs-alternate-identity?toc=%2Fvsts%2Faccounts%2Ftoc.json&bc=%2Fvsts%2Faccounts%2Fbreadcrumb%2Ftoc.json&view=vsts)
* Content Source: [subscriptions/vs-alternate-identity.md](https://github.com/Microsoft/visualstudio-docs/blob/master/subscriptions/vs-alternate-identity.md)
* Service: **unspecified**
* GitHub Login: @evanwindom
* Microsoft Alias: **jaunger**

**Comment by erickson-doug** [comment]
Hello! We will review this and update the VSTS documentation, and link to it from the VS Subscriptions docs. @chcomley @TerryGLee

**Comment by ade-taylor** [comment]
@pkskelly - So glad I came across this. This is exactly the same problem we are facing. I only stumbled across a separate article about linking Azure AD identities to the Visual Studio account as an alternate account - this was in no way clear and obvious.
I realise that Microsoft are trying to unpick the "two identities" problem, but the whole process for Partners assigning MSDN licenses and making use of VSTS is still a mess and not been resolved yet.
